### ISSUE TITLE
Add CTA for local preview under /self-hosted install now button

### ISSUE THREAD

**Issue by MichaelAring** [gitpod-io/website#2548]
**Context**
- We want to show users that they can try the local preview of self-hosted if the full installation is too complex
- This relates to [#9075](https://github.com/gitpod-io/gitpod/issues/9075)
- [Context in Notion](https://www.notion.so/gitpod/Release-Plan-Local-Preview-a635787b23a84649967a64819e6be79f#e6ce17f9a92746019b69a679e9e303d8)
- [Context in Slack](https://gitpod.slack.com/archives/C01KGM9ETU6/p1659348225921269?thread_ts=1658219260.825819&cid=C01KGM9ETU6)

**Solution**
- Add CTA "Can’t install on K8s? Try Gitpod locally with local preview" below the "Install Now" button on https://www.gitpod.io/self-hosted
- Point link to: https://github.com/gitpod-io/preview-install

**Comment by Pothulapati** [comment]
I don’t think we should be pointing to previvew-install repo, as its stale now. The code is now at https://github.com/gitpod-io/gitpod/tree/main/install/preview, while the documentation is at https://www.gitpod.io/docs/self-hosted/latest/local-preview

**Comment by lucasvaltl** [comment]
Updated the description here to say that the link should point to the docs page

**Comment by Siddhant-K-code** [comment]
Just wanted to double check this one, is it still needed?

**Comment by Siddhant-K-code** [comment]
It is resolved by #2972
